frontend/src/components/translation/TranslationTable.tsx

```diff
@@ -129,7 +134,9 @@ const TranslationTable = ({
             Translate to <strong>{translatedLanguage}</strong>
           </Text>
         </Flex>
-        <Text variant="statusHeader">Status</Text>
+        <Flex>
```

Comment:
hmm now that I think about it, shouldnt this Text be in the same flex row as the two other headers? (i.e wrapped with the flex starting at 129)